controls/V-233564.rb

```diff
@@ -69,47 +69,37 @@
   tag cci: ["CCI-000172"]
   tag nist: ["AU-12 c"]
 
-pg_conf_file= input('pg_conf_file')
+pg_conf_file= input('pg_conf_file') #not in use
 
-pg_dba = input('pg_dba')
+    sql = postgres_session(input('pg_dba'), input('pg_dba_password'), input('pg_host'), input('pg_port'))
 
-pg_dba_password = input('pg_dba_password')
-
-pg_db = input('pg_db')
-
-pg_host = input('pg_host')
-
-pg_audit_log_dir = input('pg_audit_log_dir')
-
-    sql = postgres_session(pg_dba, pg_dba_password, pg_host, input('pg_port'))
-
-  if file(pg_audit_log_dir).exist?
+  if file(input('pg_audit_log_dir')).exist?
     
-    describe sql.query('DROP TABLE IF EXISTS test_schema.test_table;', [pg_db]) do
+    describe sql.query('DROP TABLE IF EXISTS test_schema.test_table;', [input('pg_db')]) do
       its('output') { should eq 'DROP TABLE' }
     end
 
-    describe sql.query('DROP SCHEMA IF EXISTS test_schema;', [pg_db]) do
+    describe sql.query('DROP SCHEMA IF EXISTS test_schema;', [input('pg_db')]) do
       its('output') { should eq 'DROP SCHEMA' }
     end
 
-    describe sql.query('CREATE SCHEMA test_schema;', [pg_db]) do
+    describe sql.query('CREATE SCHEMA test_schema;', [input('pg_db')]) do
       its('output') { should eq 'CREATE SCHEMA' }
     end
 
-    describe sql.query('CREATE TABLE test_schema.test_table(id INT);', [pg_db]) do
+    describe sql.query('CREATE TABLE test_schema.test_table(id INT);', [input('pg_db')]) do
       its('output') { should eq 'CREATE TABLE' }
     end
     
-    describe sql.query('INSERT INTO test_schema.test_table(id) VALUES (0);', [pg_db]) do
+    describe sql.query('INSERT INTO test_schema.test_table(id) VALUES (0);', [input('pg_db')]) do
       its('output') { should eq 'INSERT 0 1' }
     end
 
-    describe sql.query('CREATE ROLE bob;', [pg_db]) do
+    describe sql.query('CREATE ROLE bob;', [input('pg_db')]) do
       its('output') { should eq 'CREATE ROLE' }
     end
 
-    describe sql.query('SET ROLE bob; SELECT * FROM test_schema.test_table;', [pg_db]) do
+    describe sql.query('SET ROLE bob; SELECT * FROM test_schema.test_table;', [input('pg_db')]) do
       its('output') { should match /ERROR:  permission denied for schema test_schema/ }
     end

```

Comment:
Looks like lines 106 and 110 need `input()` around `pg_db` @damif5

Reply:
Done